projects/storefrontapp/src/app/app.module.ts

```diff
@@ -84,7 +84,7 @@ if (!environment.production) {
     }),
     provideConfig(<FeaturesConfig>{
       features: {
-        level: '4.2',
+        level: '5.1',
```

Comment:
This soon to be merged PR should resolve that once and for all: https://github.com/SAP/spartacus/pull/16116/files